### PR TITLE
Updates dependencies

### PR DESCRIPTION
**Pull request by nobbele**

### CODE REVIEW

File: src/painter.rs [hunk 1]
```diff
@@ -197,7 +197,7 @@ impl Painter {
                             screen_size_in_pixels.1.round() as _,
                         ],
                     };
-                    callback.call(&info, ctx)
+                    callback.call(&info, unsafe { &mut ctx.static_context() })
```
Comment (not-fl3) [inline src/painter.rs:200]: Just published miniquad `0.3.8` where Context do not have any lifetimes

File: Cargo.toml [hunk 2]
```diff
@@ -17,10 +17,10 @@ keywords = ["gui", "imgui", "immediate", "portable", "gamedev"]
 include = ["../LICENSE-APACHE", "../LICENSE-MIT", "**/*.rs", "Cargo.toml"]
 
 [dependencies]
-bytemuck = "1.7"
-egui = { version = "0.18.1", features = ["bytemuck"] }
-miniquad = { version = "=0.3.0-alpha.46" }
-quad-url = "0.1.0"
+bytemuck = "1.9"
+egui = { version = "0.18", features = ["bytemuck"] }
```
Comment (emilk) [inline Cargo.toml:21]:
```suggestion
egui = { version = "0.18.1", features = ["bytemuck"] }
```

`egui-miniquad` requires `0.18.1`: https://github.com/emilk/egui/blob/master/CHANGELOG.md#0181---2022-05-01

Comment (nobbele) [reply Cargo.toml:21]: doesn't `0.18.1` get matched with `0.18` too?

Comment (emilk) [reply Cargo.toml:21]: `0.18` gets matched to either `0.18.0` or `0.18.1`, and only one of those work.

**Review by emilk — APPROVED** [review]

**Pull request merged**